compiler/src/dotty/tools/dotc/ast/untpd.scala

```diff
@@ -526,7 +526,7 @@ object untpd extends Trees.Instance[Untyped] with UntypedTreeInfo {
       case Function(args, body) =>
         this(this(x, args), body)
       case InfixOp(left, op, right) =>
-        this(this(x, left), right)
+        this(this(this(x, left), op), right)
       case PostfixOp(od, op) =>
```

Comment:
I haven't checked how `UntypedTreeAccumulator` is used exactly, but if InfixOp folds over the `op`, then surely `PostfixOp` and `PrefixOp` should too?